Graphs/Components/utils/RechartsTooltip.js

```diff
@@ -48,6 +48,18 @@ const TooltipComponent = (props) => {
                     col = payload[0];
                 }
                 let columnFormatter = columnAccessor(element);
+                if (groupedKeys && yColumn === elementKey) {
+                    return groupedKeys.map(Label => {
+                        return (
+                            <Item
+                                key={`tooltip-${index}`}
+                            >
+                                {Label} : { col['payload'][Label] !== undefined ? col['payload'][Label] && (columnFormatter(col['payload'][Label])) : col.name}
```

Comment:
why check for `col['payload'][Label]` within conditional? instead, can we have,
```
{Label} : {!isEmpty(col['payload'][Label]) ? columnFormatter(col['payload'][Label]) : col.name}
```

Reply:
@killanch 

We can not check the above condition with `isEmpty` because we have `string` and `number` types of values. isEmpty is working fine in the case of string but not working with a number.

Reply:
@nxanil agreed.
but if `col['payload'][label]` is `0`, it will still not go through columnFormatter since `col['payload'][label]` will resolve as false.
can you pls add a util function in vis-graphs similar to https://github.com/nuagenetworks/vsd-react-ui/blob/master/src/utils/index.js#L1005 and make use of same?